packages/front-end/components/Settings/MemberList.tsx

```diff
@@ -33,6 +36,7 @@ const MemberList: FC<{
       }),
     });
     mutate();
+    setRoleModal(null);
```

Comment:
I don't think you need this.  The modal automatically calls the `close` prop when `submit` finishes